bin/perseo

```diff
@@ -76,8 +76,12 @@ function loadConfiguration() {
         config.endpoint.port = process.env.PERSEO_ENDPOINT_PORT;
     }
     if (process.env.PERSEO_MONGO_HOST) {
-        config.mongo.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/cep';
-        config.orionDb.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/orion';
+        var mongoDBEndpoint = process.env.PERSEO_MONGO_HOST;
```

Comment:
Documentation should be changed consecuently according with the changes included in this PR.

Morevoer, I think that a better alterantive could be to have a PERSEO_MONGO_ENDPOINT variable (instead of PERSEO_MONGO_HOST) including the URL scheme and port, so Perseo code shouldn't deal with that. The only think that perseo code should do is to add the `/cep` and `/orion` prefixes.